### PR TITLE
Add a render_pixel option to the publisher.

### PR DESCRIPTION
**Pull request by ericholscher**
This allows us to transition to viewport tracking:

* Publisher gets send_pixel field (default True)
* Ad Type checks {% if send_pixel %} around the pixel
* We ship the client with viewport tracking
* We disable send_pixel on the users we want viewport tracking on

### CODE REVIEW

File: adserver/models.py [hunk 1]
```diff
@@ -175,6 +175,12 @@ class Publisher(TimeStampedModel, IndestructibleModel):
     record_placements = models.BooleanField(
         default=False, help_text=_("Record placement impressions for this publisher")
     )
+    render_pixel = models.BooleanField(
+        default=True,
```
Comment (ericholscher) [inline adserver/models.py:179]: Actually, I wonder if this should default False, so all new publishers need to use the client, and we can enable it if they want a custom integration. 

I think for now we default True for the migration, but eventually switch to default False

File: adserver/templates/adserver/advertisement.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <a href="{{ link_url }}" rel="nofollow" target="_blank">
       <img src="{{ image_url }}">
     </a>
-  {% elif view_url %}
+  {% elif view_url and publisher.render_pixel %}
```
Comment (davidfischer) [inline adserver/templates/adserver/advertisement.html:9]: This is only the default template that isn't used. The real one is stored in the DB.

**Review by davidfischer — APPROVED** [review]
This looks good although it might be worth it to have a test.

File: adserver/templates/adserver/advertisement.html (hunk 2 above)
Comment (ericholscher) [reply adserver/templates/adserver/advertisement.html:9]: Yea, I know -- was just an example and to keep things consistent.

**Comment by ericholscher** [comment]
Weird, that test passes for me locally -- I'm not convinced it's a useful test tho :/

**Comment by davidfischer** [comment]
> I'm not convinced it's a useful test tho :/

I think it's fine. :shipit:

**Pull request merged**